pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -187,6 +189,13 @@ func (r *RepoSyncReconciler) Reconcile(ctx context.Context, req controllerruntim
 		r.logger(ctx).V(3).Info("Sync deletion timestamp detected")
 	}
 
+	enabled, err := r.isWebhookEnabled(ctx)
+	if err != nil {
+		metrics.RecordReconcileDuration(ctx, metrics.StatusTagKey(err), start)
+		return controllerruntime.Result{}, err
```

Comment:
return controllerruntime.Result{}, fmt.Errorf("failed to get the admission webhook configuration: %w", err).